nvflare/fuel/f3/sfm/sfm_endpoint.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2021-2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+import threading
+from typing import List, Optional
+
+from ..endpoint import Endpoint
```

Comment:
maybe for parent import, we do the absolute import since, 
we could also move this SFM pacakge somewhere else?

Reply:
All changed to absolute.